.github/workflows/tasks-check.yaml

```diff
@@ -0,0 +1,20 @@
+name: Tasks Check
+
+on:
+  pull_request_review:
```

Comment:
Can we do this before giving a pull request review  somehow ?

Reply:
It could be done using the pull_request event, but using different events generates different checks on the PR. That's something that I didn't know beforehand and now I know. It's not one check per workflow. I thought using different events would rerun the same check but it creates one check per event per workflow. For example:

- Tasks Check (pull_request/opened)
- Tasks Check (pull_request_review/approved)

If I added the pull_request event, It would generate the Tasks Check (pull_request/opened) and this one would only be updated if the user pushed something into the branch. Which may not happen and the PR would be forever stuck.

That's why I decided to run it after the PR approval.